Assets/VRTK/SDK/Daydream/DaydreamControllerLog.cs

```diff
@@ -0,0 +1,169 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+
+public class DaydreamControllerLog : MonoBehaviour
+{
+    public bool consoleLogging;
+    public bool csvLogging;
+    public string csvPath = @"d:\ControllerData.csv";
+
+    private StreamWriter writer;
+
+    void Start()
```

Comment:
Also, would it better to do this in `OnEnable` then you could determine if `csvLogging` is required every time the script is enabled, rather than just once at start.